<html><head>
<title>README</title></head>
<body BGCOLOR="dddddd" LINK="darkgreen" VLINK="darkblue" ALINK="darkred">
<h1>What should you expect to see in COMe?</h1>
<h2>A guide to the COMe data in XML</h2>

<table border="0" cellpadding="5">
<tr>
<td align=right>
</td>
</tr>
</table>
<p align=right>
<i>Created: 5 April 2001</i>
<br><i>Last modified: 20 April 2004</i>
</p>
<hr>
<ol>
<a href="#intro">Introduction</a>

<li><a href="#molecule">Molecule</a>
<ul>
<a href="#1.1">Example <b>1.1</b></a>
</ul>
</li>
<li><a href="#BIM">Bioinorganic motif</a>
<ul>
<a href="#2.1">Example <b>2.1</b>: mononuclear metal centre</a>
<br><a href="#2.2">Example <b>2.2</b>: mononuclear metal centre with didentate
binding</a>
<br><a href="#2.3">Example <b>2.3</b>: dinuclear metal centre</a>
<br><a href="#2.4">Example <b>2.4</b>: polynuclear metal centre</a>
<br><a href="#2.5">Examples <b>2.5</b> and <b>2.6</b>: 
metal-tetrapyrrole complex</a>
<br><a href="#2.7">Example <b>2.7</b>: metal-molybdopterin complex</a>
<br><a href="#2.8">Example <b>2.8</b>: organic prosthetic group</a>
</ul>
</li>

<li><a href="#protein">Protein</a>
<ul>
<a href="#3.1">Example <b>3.1</b></a>
</ul></li>
<li><a href="#relationships">Relationships in ontology</a>
<ul>
<a href="#KindOf">KindOf relationship</a>
<ul>
<a href="#4.1">Example <b>4.1</b>: KindOf relationship for PRX</a>
<br><a href="#4.2">Example <b>4.2</b>: child with many parents</a>
<br><a href="#4.3">Example <b>4.3</b>: KindOf relationship for BIM</a>
<br><a href="#4.4">Example <b>4.4</b>: KindOf relationship for MOL</a>
</ul>
<a href="#PartOf">PartOf relationship</a>
<ul>
<a href="#4.5">Example <b>4.5</b>: PartOf relationship for BIM</a>
</ul>
<a href="#isBoundTo">isBoundTo relationship</a>
<ul>
<a href="#4.6">Example <b>4.6</b>: isBoundTo relationship</a>
</ul>
<a href="#4.7">Table <b>4.7</b>: The summary of relationships</a>
</ul>
</li>
<li><a href="#references">References</a></li>
</ol>
<hr>
<p>

<a name="intro">
<h2>Introduction</h2>

The experimental ontology for bioinorganic centres, aka COMe, is currently 
available from 
<a href="http://www.ebi.ac.uk/~kirill/come/COME.xml">http://www.ebi.ac.uk/~kirill/come/COME.xml</a>
and could be navigated with Microsoft Internet Explorer (version 5+).
</p><p>
COMe consists of three types of entities: Molecule (MOL), 
Bioinorganic Motif (BIM), and Bioinorganic Proteins (PRX). 
The structure of these three file types is explained in subsequent sections.
</p><p>
XML was chosen because it is well suited to represent hierarchies and is
easy to read with any text editor.
Currently, XML is used to edit and enter new entries; then they are downloaded
into relational database.
</p>

<a name="molecule">
<h2>1. Molecule</h2>

<p>
Both Molecule (MOL) and Bioinorganic Motif (BIM) entities consist of 
<font face="Arial"><font color="darkred">substructure</font></font> elements.
"Substructure" literally means that these elements can form parts of a bigger
structure, i.e. complete protein (but see <a href="#isBoundTo">isBoundTo
relationship</a>). 
MOL is an entity representing "small molecule" (as opposed to macromolecule)
which, in complex with polypeptide, forms a functional protein.
</p>
<a name="1.1">
<table width="100%" border="0" cellpadding="5">
<tr><td bgcolor="white"><font face="Arial" color="blue">
<ul>
&#60;<font color="darkred">substructure value</font>="<font color="black"><b>bacteriochlorophyll a</b></font>" <font color="darkred">id</font>="<font color="black"><b>MOL000001</b></font>"&#62;
  <ul>&#60;<font color="darkred">term value</font>="<font color="black"><b>bacteriochlorophyll a</b></font>" <font color="darkred">dbxref</font>="<font color="black"><b>PDBHET:BCL</b></font>" /&#62;
</ul>&#60;/<font color="darkred">substructure</font>&#62;
</ul></font></td>
<td align=right><b>(1.1)</b></td></tr>
</table>

<p>
As you can see, there are (yet?) no systematic names or chemical formulae. 
Instead, the MOL entities (sometimes) contain cross-references to 
chemical and/or crystallography databases 
(<font face="Arial"><font color="darkred">dbxref</font></font>) or cross-references to the 
bibliography databases (<font face="Arial"><font color="darkred">lref</font></font>).
The databases currently cross-referenced are:
COMPOUND (component of 
<a href="http://www.genome.ad.jp/dbget/ligand.html">LIGAND</a> database),
<a href="http://www.biochem.ucl.ac.uk/bsm/pdbsum/hetgroups/">PDBHET</a> (HET 
groups derived from PDB) and
<a href="http://srs.ebi.ac.uk/srsbin/cgi-bin/wgetz?-page+LibInfo+-lib+RESID">RESID</a>.
The bibliography database referenced to in MOL, BIM and PRX entries is MEDLINE.
</p>

<a name="BIM">
<h2>2. Bioinorganic motif</h2>

<p>
The definition of Bioinorganic Motif (BIM) is given elsewhere
[<a href="#ref1">1</a>, <a href="#ref2">2</a>].
In COMe representation, every BIM consists of at least one
<a href="centres.html">centre</a> and two or more
<a href="ligands.html">ligands</a>.
<a name="2.1">
For <b>mononuclear</b> metal centres, BIMs look like this:
</p>
<table width="100%" border="0" cellpadding="5">
<tr><td bgcolor="white"><font face="Arial" color="blue">
<ul>
&#60;<font color="darkred">substructure id</font>="<font color="black"><b>BIM000165</b></font>"&#62;
  <ul>&#60;<font color="darkred">bim coordination</font>="<font color="black"><b>TBPY-5</b></font>"&#62;<font color="black"><b>Zn(NE.His)2(OD.Asp)(OH2)2</b></font>&#60;/<font color="darkred">bim</font>&#62;
</ul>&#60;/<font color="darkred">substructure</font>&#62;</ul></font></td>
<td align=right><b>(2.1)</b></td></tr>
</table>

<p>
The <font face="Arial"><font color="darkred">bim</font></font> element includes
a BIM itself.
The rule for writing a mononuclear centre BIM: first comes the metal, then 
endogenous ligands (amino acid residues), then exogenous ligands, e.g. water
(see the <a href="ligands.html">list of ligands</a>).
This is a pretty simplistic description, it does not take into account 
stereochemistry at the metal atom.
The <font face="Arial"><font color="darkred">coordination</font></font> attribute is not 
mandatory for it may be unknown. It also does not make sense for polynuclear metal
centres.

In the example <b>(2.1)</b>, one zinc atom is coordinated by 
N<sup><font face="symbol">e</font></sup> atoms of two His residues, one 
O<sup><font face="symbol">d</font></sup> atom of Asp residue and two water 
molecules. The coordination geometry is trigonal bipyramidal 
(<font face="Arial"><b>TBPY-5</b></font>; see the <a href="polyhedra.html">list
of coordination polyhedra</a>).
</p>
<a name="2.2">
<table width="100%" border="0" cellpadding="5">
<tr><td bgcolor="white"><font face="Arial" color="blue">
<ul>
&#60;<font color="darkred">substructure id</font>="<font color="black"><b>BIM000176</b></font>"&#62;
  <ul>&#60;<font color="darkred">bim coordination</font>="<font color="black"><b>T-4</b></font>"&#62;<font color="black"><b>Zn(ND.His)2{k2-(O,SG.Cys)}</b></font>&#60;/<font color="darkred">bim</font>&#62;
</ul>&#60;/<font color="darkred">substructure</font>&#62;</ul></font></td>
<td align=right><b>(2.2)</b></td></tr>
</table>

<p>
In the example <b>(2.2)</b>, the zinc atom is tetrahedrally 
(<font face="Arial"><b>T-4</b></font>) coordinated by N<sup><font face="symbol">d</font></sup> 
atoms of two His residues and by two atoms of Cys residue, 
S<sup><font face="symbol">g</font></sup> and mainchain oxygen
(<font face="Arial"><b>k2</b></font> prefix designates didentate binding).
</p><p><a name="2.3">
For <b>dinuclear</b> metal centres, BIMs look like this:
</p>

<table width="100%" border="0" cellpadding="5">
<tr><td bgcolor="white"><font face="Arial" color="blue">
<ul>
&#60;<font color="darkred">substructure id</font>="<font color="black"><b>BIM000104</b></font>"&#62;
  <ul>&#60;<font color="darkred">bim</font>&#62;<font color="black"><b>{Cu(ND.His)(SD.Met)}{Cu(ND.His)(O.Glu)}{&#181;-(SG.Cys)}2</b></font>&#60;/<font color="darkred">bim</font>&#62;
</ul>&#60;/<font color="darkred">substructure</font>&#62;</ul></font></td>
<td align=right><b>(2.3)</b></td></tr>
</table>
<p>
where every metal atom and its unique ligands are enclosed in braces and
<font face="Arial"><b>&#181;</b></font> prefix designates bridging ligands.
</p>
<p>
<a name="2.4">
For <b>polynuclear</b> metal centres, BIMs look like this:
</p>
<table width="100%" border="0" cellpadding="5">
<tr><td bgcolor="white"><font face="Arial" color="blue">
<ul>
&#60;<font color="darkred">substructure id</font>="<font color="black"><b>BIM000059</b></font>"&#62;
  <ul>&#60;<font color="darkred">bim</font>&#62;<font color="black"><b>(Fe4S4)(NE.His)(SG.Cys)3</b></font>&#60;/<font color="darkred">bim</font>&#62;
  <br>&#60;<font color="darkred">term dbxref</font>="<font color="black"><b>RESID:AA0284</b></font>" /&#62;</br>
</ul>&#60;/<font color="darkred">substructure</font>&#62;</ul></font></td>
<td align=right><b>(2.4)</b></td></tr>
</table>
<p>
For simplicity, the cubane iron-sulfur unit is represented as 
<font face="Arial"><b>(Fe4S4)</b></font>
and not <font face="Arial"><b>{Fe4(&#181;3-S)4}</b></font>.
</p>

<p><a name="2.5">
The rule for writing a BIM for centre containing <b>metal-organic group 
complex</b>: 
first comes the metal, then organic group, then amino acid residues,
then exogenous ligands (e.g. CO).
For metal-tetrapyrrole (heme, chlorophyll, cobalamin) proteins, BIMs look
like this:
</p>
<table width="100%" border="0" cellpadding="5">
<tr><td bgcolor="white"><font face="Arial" color="blue">
<ul>
&#60;<font color="darkred">substructure id</font>="<font color="black"><b>BIM000209</b></font>"&#62;
  <ul>&#60;<font color="darkred">bim coordination</font>="<font color="black"><b>SPY-5</b></font>"&#62;<font color="black"><b>Co(Crn)(NE.His)</b></font>&#60;/<font color="darkred">bim</font>&#62;
</ul>&#60;/<font color="darkred">substructure</font>&#62;</ul></font></td>
<td align=right><b>(2.5)</b></td></tr>
<tr><td bgcolor="white"><font face="Arial" color="blue">
<ul>
&#60;<font color="darkred">substructure id</font>="<font color="black"><b>BIM000212</b></font>"&#62;
  <ul>&#60;<font color="darkred">bim coordination</font>="<font color="black"><b>OC-6</b></font>"&#62;<font color="black"><b>Fe(por)(SG.Cys)(CO)</b></font>&#60;/<font color="darkred">bim</font>&#62;
</ul>&#60;/<font color="darkred">substructure</font>&#62;</ul></font></td>
<td align=right><b>(2.6)</b></td></tr>
</table>
<p>

where <font face="Arial"><b>Crn</b></font> = corrin ligand, <font face="Arial"><b>por</b></font> = porphyrin 
ligand. These ligands are assumed always to be tetradentate, that's why 
I do not write <font face="Arial"><b>{k4-(N,N,N,N.por)}</b></font>. 
<a name="2.7">
For molybdopterin-containing proteins, BIMs look like this:
</p>
<table width="100%" border="0" cellpadding="5">
<tr><td bgcolor="white"><font face="Arial" color="blue">
<ul>
&#60;<font color="darkred">substructure id</font>="<font color="black"><b>BIM000181</b></font>"&#62;
  <ul>&#60;<font color="darkred">bim coordination</font>="<font color="black"><b>TPR-6</b></font>"&#62;<font color="black"><b>Mo{k2-(S1',S2'.molybdopterin)}2(SeG.Sec)(OH)</b></font>&#60;/<font color="darkred">bim</font>&#62;
</ul>&#60;/<font color="darkred">substructure</font>&#62;</ul></font></td>
<td align=right><b>(2.7)</b></td></tr>
</table>
<p>
<a name="2.8">
Finally, for a purely <b>organic prosthetic group</b> covalently attached to 
the polypeptide, MOL description is not enough and BIM files are used, although
without metal:
</p>
<table width="100%" border="0" cellpadding="5">
<tr><td bgcolor="white"><font face="Arial" color="blue">
<ul>
&#60;<font color="darkred">substructure id</font>="<font color="black"><b>BIM000201</b></font>"&#62;
  <ul>&#60;<font color="darkred">bim</font>&#62;<font color="black"><b>(C8M.FAD)(SG.Cys)</b></font>&#60;/<font color="darkred">bim</font>&#62;
  <br>&#60;<font color="darkred">term value</font>="<font color="black"><b>8alpha-(S-cysteinyl)FAD</b></font>" <font color="darkred">dbxref</font>="<font color="black"><b>RESID:AA0143</b></font>" /&#62;</br>
</ul>&#60;/<font color="darkred">substructure</font>&#62;</ul></font></td>
<td align=right><b>(2.8)</b></td></tr>
</table>

<a name="protein">
<h2>3. Protein</h2>

<p> 
The <b>protein</b> (PRX) entity refers to the functional protein as well as
separate protein domains and subunits. A typical low-level PRX entity is 
shown below:
</p>
<a name="3.1">
<table width="100%" border="0" cellpadding="5">
<tr><td bgcolor="white"><font face="Arial" color="blue">
<ul>
&#60;<font color="darkred">term value</font>="<font color="black"><b>NADPH:P450 reductase</b></font>" <font color="darkred">id</font>="<font color="black"><b>PRX000111</b></font>" <font color="darkred">dbxref</font>="<font color="black"><b>EC:1.6.2.4</b></font>"&#62;
 <ul>&#60;<font color="darkred">instance value</font>="<font color="black"><b>NADPH:P450 reductase</b></font>" <font color="darkred">species</font>="<font color="black"><b>rat</b></font>" <font color="darkred">dbxref</font>="<font color="black"><b>PDB:1AMO</b></font>" /&#62;
</ul>&#60;/<font color="darkred">term</font>&#62;</ul></font></td>
<td align=right><b>(3.1)</b></td></tr>
</table>
<p>
The PRX entity minimally includes the 
<font face="Arial"><font color="darkred">term</font></font> element with 
mandatory attributes 
<font face="Arial"><font color="darkred">value</font></font> and
<font face="Arial"><font color="darkred">id</font></font>. 
It also may include 
<font face="Arial"><font color="darkred">instance</font></font> elements as 
well as MOL, BIM and other PRX entities.
The <font face="Arial"><font color="darkred">instance</font></font> element has
mandatory attributes 
<font face="Arial"><font color="darkred">value</font></font> and
<font face="Arial"><font color="darkred">species</font></font>. 
The role of <font face="Arial"><font color="darkred">instance</font></font> is
to provide the external evidence that the protein in question does exist. 
There is no <font face="Arial"><font color="darkred">id</font></font> 
attribute for <font face="Arial"><font color="darkred">instance</font></font> 
(yet?) but either <font face="Arial"><font color="darkred">dbxref</font></font>
or <font face="Arial"><font color="darkred">lref</font></font> have to be 
provided (while they are not always available for the parent
<font face="Arial"><font color="darkred">term</font></font>). 
The databases currently cross-referenced are: 
<a href="http://www.flymine.org/swissprot/">SPTR</a>
(SWISS-PROT and TrEMBL), <a href="http://pdb-browsers.ebi.ac.uk/">PDB</a>,
<a href="http://www.flymine.org/interpro/">InterPro</a>, EC
(<a href="http://www.expasy.ch/enzyme/">ENZYME</a>),
<a href="http://www.merops.co.uk/">MEROPS</a> and
<a href="http://genome-www.stanford.edu/GO/">GO</a>.
The other attributes are
<font face="Arial"><font color="darkred">centre</font></font> (for proteins containing more than
one bioinorganic centre) and <font face="Arial"><font color="darkred">state</font></font>.
</p>
<a name="relationships">
<h2>4. Relationships in ontology</h2>
<p>
The relationship between entities are not made explicit in XML, but can be
deduced using the set of rules. The relational implementation, however,
provides explicit relationships. 
</p>
<a name="KindOf">
<h3>KindOf</h3>

<p>The example below shows the fragment of COMe ontology (where 
"<font face="Arial" color="red">+</font>" means that everything in hierarchy 
below this particular level is hidden; 
"<font face="Arial" color="red">-</font>" means that <i>not</i> everything is 
hidden):
</p>
<a name="4.1">
<table width="100%" border="0" cellpadding="5">
<tr><td bgcolor="white"><font face="Arial" color="blue">
<ul><font color="red">-</font> &#60;<font color="darkred">term value</font>="<font color="black"><b>complex protein</b></font>" <font color="darkred">id</font>="<font color="black"><b>PRX000001</b></font>"&#62;
  <ul><font color="red">-</font> &#60;<font color="darkred">term value</font>="<font color="black"><b>metalloprotein</b></font>" <font color="darkred">id</font>="<font color="black"><b>PRX000002</b></font>"&#62;
    <ul><font color="red">+</font> &#60;<font color="darkred">term value</font>="<font color="black"><b>calcium protein</b></font>" <font color="darkred">id</font>="<font color="black"><b>PRX000482</b></font>"&#62;
    <br><font color="red">+</font> &#60;<font color="darkred">term value</font>="<font color="black"><b>cadmium protein</b></font>" <font color="darkred">id</font>="<font color="black"><b>PRX000664</b></font>"&#62;
    <br><font color="red">+</font> &#60;<font color="darkred">term value</font>="<font color="black"><b>cobalt protein</b></font>" <font color="darkred">id</font>="<font color="black"><b>PRX000631</b></font>"&#62;
    <br><font color="red">+</font> &#60;<font color="darkred">term value</font>="<font color="black"><b>copper protein</b></font>" <font color="darkred">id</font>="<font color="black"><b>PRX000009</b></font>"&#62;
    <br><font color="red">-</font> &#60;<font color="darkred">term value</font>="<font color="black"><b>iron protein</b></font>" <font color="darkred">id</font>="<font color="black"><b>PRX000004</b></font>"&#62;
      <ul><font color="red">+</font> &#60;<font color="darkred">term value</font>="<font color="black"><b>mononuclear iron protein</b></font>" <font color="darkred">id</font>="<font color="black"><b>PRX000005</b></font>"&#62;
      <br><font color="red">+</font> &#60;<font color="darkred">term value</font>="<font color="black"><b>diiron-carboxylate protein</b></font>" <font color="darkred">id</font>="<font color="black"><b>PRX000006</b></font>"&#62;
      <br><font color="red">+</font> &#60;<font color="darkred">term value</font>="<font color="black"><b>Fe,Ni protein</b></font>" <font color="darkred">id</font>="<font color="black"><b>PRX000082</b></font>"&#62;
      <br><font color="red">+</font> &#60;<font color="darkred">term value</font>="<font color="black"><b>iron-sulfur protein</b></font>" <font color="darkred">id</font>="<font color="black"><b>PRX000007</b></font>"&#62;
        <ul><font color="red">+</font> &#60;<font color="darkred">term value</font>="<font color="black"><b>Fe2S2 protein</b></font>" <font color="darkred">id</font>="<font color="black"><b>PRX000053</b></font>"&#62;
        <br><font color="red">+</font> &#60;<font color="darkred">term value</font>="<font color="black"><b>Fe(3|4)S4 protein</b></font>" <font color="darkred">id</font>="<font color="black"><b>PRX000054</b></font>"&#62;
        <br><font color="red">+</font> &#60;<font color="darkred">term value</font>="<font color="black"><b>P-cluster protein</b></font>" <font color="darkred">id</font>="<font color="black"><b>PRX000055</b></font>"&#62;
        <br><font color="red">+</font> &#60;<font color="darkred">term value</font>="<font color="black"><b>M-cluster protein</b></font>" <font color="darkred">id</font>="<font color="black"><b>PRX000056</b></font>" <font color="darkred">dbxref</font>="<font color="black"><b>InterPro:IPR000318</b></font>"&#62;
        <br><font color="red">+</font> &#60;<font color="darkred">term value</font>="<font color="black"><b>hybrid cluster protein</b></font>" <font color="darkred">id</font>="<font color="black"><b>PRX000057</b></font>"&#62;
      </ul><font color="red">+</font> &#60;<font color="darkred">term value</font>="<font color="black"><b>heme protein</b></font>" <font color="darkred">id</font>="<font color="black"><b>PRX000008</b></font>"&#62;</font>
</ul></ul></ul>...</ul></td>
<td align=right><b>(4.1)</b></td></tr>
</table>
<p>
Example <b>(4.1)</b> illustrates the <b>KindOf</b> (child to parent) 
relationship (also known as <b>ISA</b> relationship).
<font face="Arial"><b>Fe2S2 protein</b></font> is kind of <font face="Arial"><b>iron-sulfur protein</b></font>
which is kind of <font face="Arial"><b>iron protein</b></font> which is kind of 
<font face="Arial"><b>metalloprotein</b></font>.
This relationship occurs between entities of the same class (PRX to PRX, 
BIM to BIM, MOL to MOL).</p>
<p>
An important feature of this relationship is <b>inheritance</b>. For example, 
all proteins belonging to <font face="Arial"><b>Fe2S2 protein</b></font> class will inherit
substructure <font face="Arial"><b>(Fe2S2)(SG.Cys)2*</b></font> (BIM000063).
</p>
<p>
<a name="4.2">
An entity may have more than one parent, as example <b>(4.2)</b> shows:
</p>

<center>
<img SRC="http://www.flymine.org/come/newdags/PRX000559.gif" USEMAP=#damap><map name="damap">
<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX000001" title="PRX000001" alt="complex protein" coords="343,77,431,125">
<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX000002" title="PRX000002" alt="metalloprotein" coords="272,165,352,213">
<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX000230" title="PRX000230" alt="organic prosthetic group protein" coords="361,165,527,213">
<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX000004" title="PRX000004" alt="iron protein" coords="205,253,277,301">
<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX000102" title="PRX000102" alt="metal-pyranopterin protein" coords="317,488,461,536">
<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX000599" title="PRX000599" alt="molybdenum protein" coords="256,312,368,360">
<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX001307" title="PRX001307" alt="dinuclear protein" coords="83,253,173,301">
<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX000007" title="PRX000007" alt="iron-sulphur protein" coords="157,429,267,477">

<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX000053" title="PRX000053" alt="Fe2S2 protein" coords="101,576,181,624">
<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX000011" title="PRX000011" alt="pyranopterin protein" coords="388,370,500,418">
<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX000016" title="PRX000016" alt="FAD protein" coords="513,370,585,418">
<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX000588" title="PRX000588" alt="vanillyl-alcohol oxidase-like flavoprotein" coords="467,488,675,536">
<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX000058" title="PRX000058" alt="Fe2S2 ferredoxin" coords="53,693,144,741">
<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX000598" title="PRX000598" alt="four-helix bundle Fe2S2 domain" coords="140,752,303,800">
<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX000963" title="PRX000963" alt="plant and mammalian-type ferredoxin" coords="7,810,199,858">
<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX000062" title="PRX000062" alt="plant-type ferredoxin domain" coords="52,928,204,976">
<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX000066" title="PRX000066" alt="xanthine oxidase-like Fe2S2 protein" coords="131,1016,312,1064">
<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX000596" title="PRX000596" alt="carbon monoxide oxidase, iron-sulphur subunit" coords="155,1104,389,1152">
<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX001183" title="PRX001183" alt="molybdenum-pyranopterin protein" coords="300,576,479,624">
<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX000269" title="PRX000269" alt="flavoprotein" coords="496,253,568,301">
<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX000429" title="PRX000429" alt="pterin protein" coords="405,253,483,301">
<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX000559" title="PRX000559" alt="carbon monoxide oxidase" coords="352,1192,480,1240">
<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX001084" title="PRX001084" alt="Cu-S-Mo carbon monoxide oxidase" coords="243,1280,421,1328">
<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX001085" title="PRX001085" alt="Mo carbon monoxide oxidase" coords="427,1280,576,1328">
<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX000595" title="PRX000595" alt="carbon monoxide oxidase, flavoprotein subunit" coords="444,634,679,682">

<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX000594" title="PRX000594" alt="carbon monoxide oxidase, catalytic subunit" coords="308,869,524,917">
<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX000597" title="PRX000597" alt="xanthine oxidase-like pyranopterin protein" coords="308,752,524,800">
<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX001306" title="PRX001306" alt="homodinuclear protein" coords="59,370,179,418">
<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX001308" title="PRX001308" alt="dinuclear iron protein" coords="59,488,173,536">
<area shape="rect" href="http://www.flymine.org/come/entry?gn=PRX000559" alt="PRX000559" coords="7,5,679,1338">
</map>
</center>

<p>
In the example <b>(4.2)</b>, carbon monoxide oxidase inherits the features from
each subunit, viz. two different Fe<sub>2</sub>S<sub>2</sub> clusters, 
molybdenum cofactor (Mo-molybdopterin complex) and FAD.
</p>

<p>
<a name="4.3">
Examples <b>(4.3)</b> and <b>(4.4)</b> show the <b>KindOf</b> relationship
used to create ontologies of BIM and MOL entities. 
Note the asterisk in <font face="Arial" color="darkred">bim</font>! It is used
at this moment to distinguish <b>KindOf</b> and
<a href="#PartOf"><b>PartOf</b></a> relationships
which both could occur between BIM entries.
</p>
<table width="100%" border="0" cellpadding="5">
<tr><td bgcolor="white"><font face="Arial" color="blue">
<ul>
&#60;<font color="darkred">substructure id</font>="<font color="black"><b>BIM000025</b></font>"&#62;
  <ul>&#60;<font color="darkred">bim</font>&#62;<font color="black"><b>Fe(por)(NE.His)*</b></font>&#60;/<font color="darkred">bim</font>&#62;
  <br>&#60;<font color="darkred">substructure id</font>="<font color="black"><b>BIM000007</b></font>"&#62;
  <ul>&#60;<font color="darkred">bim coordination</font>="<font color="black"><b>SPY-5</b></font>"&#62;<font color="black"><b>Fe(por)(NE.His)</b></font>&#60;/<font color="darkred">bim</font>&#62;
  </ul>&#60;/<font color="darkred">substructure</font>&#62;
  <br>&#60;<font color="darkred">substructure id</font>="<font color="black"><b>BIM000018</b></font>"&#62;
  <ul>&#60;<font color="darkred">bim coordination</font>="<font color="black"><b>OC-6</b></font>"&#62;<font color="black"><b>Fe(por)(NE.His)(NO)</b></font>&#60;/<font color="darkred">bim</font>&#62;
  </ul>&#60;/<font color="darkred">substructure</font>&#62;
  <br>&#60;<font color="darkred">substructure id</font>="<font color="black"><b>BIM000019</b></font>"&#62;
  <ul>&#60;<font color="darkred">bim coordination</font>="<font color="black"><b>OC-6</b></font>"&#62;<font color="black"><b>Fe(por)(NE.His)(OH)</b></font>&#60;/<font color="darkred">bim</font>&#62;
  </ul>&#60;/<font color="darkred">substructure</font>&#62;
  <br>&#60;<font color="darkred">substructure id</font>="<font color="black"><b>BIM000021</b></font>"&#62;
  <ul>&#60;<font color="darkred">bim coordination</font>="<font color="black"><b>OC-6</b></font>"&#62;<font color="black"><b>Fe(por)(NE.His)(O2)</b></font>&#60;/<font color="darkred">bim</font>&#62;
  </ul>&#60;/<font color="darkred">substructure</font>&#62;
  <br>&#60;<font color="darkred">substructure id</font>="<font color="black"><b>BIM000022</b></font>"&#62;
  <ul>&#60;<font color="darkred">bim coordination</font>="<font color="black"><b>OC-6</b></font>"&#62;<font color="black"><b>Fe(por)(NE.His)(CO2)</b></font>&#60;/<font color="darkred">bim</font>&#62;
  </ul>&#60;/<font color="darkred">substructure</font>&#62;
  <br>&#60;<font color="darkred">substructure id</font>="<font color="black"><b>BIM000023</b></font>"&#62;
  <ul>&#60;<font color="darkred">bim coordination</font>="<font color="black"><b>OC-6</b></font>"&#62;<font color="black"><b>Fe(por)(NE.His)(CO)</b></font>&#60;/<font color="darkred">bim</font>&#62;
  </ul>&#60;/<font color="darkred">substructure</font>&#62;
  <br>&#60;<font color="darkred">substructure id</font>="<font color="black"><b>BIM000024</b></font>"&#62;
  <ul>&#60;<font color="darkred">bim coordination</font>="<font color="black"><b>OC-6</b></font>"&#62;<font color="black"><b>Fe(por)(NE.His){(O2)-}</b></font>&#60;/<font color="darkred">bim</font>&#62;
  </ul>&#60;/<font color="darkred">substructure</font>&#62;
  <br>&#60;<font color="darkred">substructure id</font>="<font color="black"><b>BIM000198</b></font>"&#62;
  <ul>&#60;<font color="darkred">bim coordination</font>="<font color="black"><b>OC-6</b></font>"&#62;<font color="black"><b>Fe(por)(NE.His)(CN)</b></font>&#60;/<font color="darkred">bim</font>&#62;
  </ul>&#60;/<font color="darkred">substructure</font>&#62;
</ul>&#60;/<font color="darkred">substructure</font>&#62;</ul></font></td>
<td align=right><b>(4.3)</b></td></tr>
</table>
<p>
<a name="4.4">
Similarly, the <b>KindOf</b> relationship could be applied to the MOL 
entities <b>(4.4)</b>, with the difference that asterisk is now found in
<font face="Arial" color="darkred">substructure value</font>:
</p>
<table width="100%" border="0" cellpadding="5">
<tr><td bgcolor="white"><font face="Arial" color="blue">
<ul>
&#60;<font color="darkred">substructure value</font>="<font color="black"><b>FAD*</b></font>" <font color="darkred">id</font>="<font color="black"><b>MOL000041</b></font>"&#62;
  <ul>&#60;<font color="darkred">term value</font>="<font color="black"><b>flavin adenine dinucleotide</b></font>" /&#62;
  <br>&#60;<font color="darkred">substructure value</font>="<font color="black"><b>FAD</b></font>" <font color="darkred">id</font>="<font color="black"><b>MOL000039</b></font>"&#62;
  <ul>&#60;<font color="darkred">term value</font>="<font color="black"><b>flavin-adenine dinucleotide</b></font>" <font color="darkred">dbxref</font>="<font color="black"><b>PDBHET:FAD</b></font>" /&#62;
  <br>&#60;<font color="darkred">term value</font>="<font color="black"><b>FAD</b></font>" <font color="darkred">dbxref</font>="<font color="black"><b>COMPOUND:C00016</b></font>" /&#62;
  </ul>&#60;/<font color="darkred">substructure</font>&#62;
  <br>&#60;<font color="darkred">substructure value</font>="<font color="black"><b>FADH2</b></font>" <font color="darkred">id</font>="<font color="black"><b>MOL000040</b></font>"&#62;
  <ul>&#60;<font color="darkred">term value</font>="<font color="black"><b>dihydroflavine-adenine dinucleotide</b></font>" <font color="darkred">dbxref</font>="<font color="black"><b>PDBHET:FDA</b></font>" /&#62;
  <br>&#60;<font color="darkred">term value</font>="<font color="black"><b>FADH2</b></font>" <font color="darkred">dbxref</font>="<font color="black"><b>COMPOUND:C01352</b></font>" /&#62;
  </ul>&#60;/<font color="darkred">substructure</font>&#62;
  <br>&#60;<font color="darkred">substructure value</font>="<font color="black"><b>FADH&#183;</b></font>" <font color="darkred">id</font>="<font color="black"><b>MOL000042</b></font>"&#62;
  <ul>&#60;<font color="darkred">term value</font>="<font color="black"><b>flavin adenine dinucleotide semiquinone radical</b></font>" /&#62;
  </ul>&#60;/<font color="darkred">substructure</font>&#62;
</ul>&#60;/<font color="darkred">substructure</font>&#62;</ul></font></td>
<td align=right><b>(4.4)</b></td></tr>
</table>

<a name="PartOf">
<h3>PartOf (aggregation)</h3>

<p>
This relationship can occur between entities of the same class (BIM to BIM,
MOL to MOL) or different classes (MOL to BIM, BIM to PRX).
Example <b>(4.5)</b> illustrates the <b>PartOf</b> relationship (BIM to BIM).
</p>
<a name="4.5">
<table width="100%" border="0" cellpadding="5">
<tr><td bgcolor="white"><font face="Arial" color="blue">
<ul>
&#60;<font color="darkred">substructure id</font>="<font color="black"><b>BIM000204</b></font>"&#62;
  <ul>&#60;<font color="darkred">bim</font>&#62;<font color="black"><b>{Co(NE.His)(OE.Glu)}[Co{k2-(OD,OD.Asp)}(OH2)]{&#181;-(OD,OD.Asp)}{&#181;-(OE,OE.Glu)}{&#181;-(OH2)}</b></font>&#60;/<font color="darkred">bim</font>&#62;
  <br>&#60;<font color="darkred">substructure id</font>="<font color="black"><b>BIM000205</b></font>"&#62;
  <ul>&#60;<font color="darkred">bim coordination</font>="<font color="black"><b>TBPY-5</b></font>"&#62;<font color="black"><b>Co(NE.His)(OD.Asp)(OE.Glu)2(OH2)</b></font>&#60;/<font color="darkred">bim</font>&#62;
  </ul>&#60;/<font color="darkred">substructure</font>&#62;
  <br>&#60;<font color="darkred">substructure id</font>="<font color="black"><b>BIM000206</b></font>"&#62;
  <ul>&#60;<font color="darkred">bim coordination</font>="<font color="black"><b>OC-6</b></font>"&#62;<font color="black"><b>Co(OD.Asp){k2-(OD,OD.Asp)}(OE.Glu)(OH2)2</b></font>&#60;/<font color="darkred">bim</font>&#62;
  </ul>&#60;/<font color="darkred">substructure</font>&#62;
</ul>&#60;/<font color="darkred">substructure</font>&#62;</ul></font></td>
<td align=right><b>(4.5)</b></td></tr>
</table>
<p>
In this example, each of the two mononuclear centres is part of the
dinuclear metal centre. Note that for the mononuclear centres it is possible
to indicate their coordination geometry. On the other hand, 
<font face="Arial"><b>BIM000204</b></font> is not just a sum of
<font face="Arial"><b>BIM000205</b></font> and 
<font face="Arial"><b>BIM000206</b></font>. 
Nothing in <font face="Arial"><b>BIM000205</b></font> and 
<font face="Arial"><b>BIM000206</b></font> indicates which ligands bridge the
two metal atoms.
</p>

<a name="isBoundTo">
<h3>isBoundTo</h3>

<p>
This special relationship can occur only in one case: MOL to PRX. 
The reason to use it that the molecule which <b>isBoundTo</b> protein can be
changed chemically and, strictly speaking, become the different entity.
For instance <b>(4.6)</b>, we can say that the protein binds pyridoxal
5'-phosphate (<font face="Arial"><b>MOL000108</b></font>)
but the resulting substructure has no aldehyde group and therefore
is different from "free" pyridoxal 5'-phosphate, so pyridoxal 5'-phosphate is
<b><i>not</i></b> a <b>PartOf</b> "pyridoxal 5'-phosphate protein".
In contrast, <font face="Arial"><b>BIM000270</b></font> is <b>PartOf</b> that
protein. 

</p>
<a name="4.6">
<table width="100%" border="0" cellpadding="5">
<tr><td bgcolor="white">
 <font face="Arial" color="blue">
<ul>
&#60;<font color="darkred">term value</font>="<font color="black"><b>pyridoxal 5'-phosphate protein</b></font>" <font color="darkred">id</font>="<font color="black"><b>PRX000808</b></font>"&#62;
 <ul>&#60;<font color="darkred">substructure value</font>="<font color="black"><b>pyridoxal 5'-phosphate</b></font>" <font color="darkred">id</font>="<font color="black"><b>MOL000108</b></font>"&#62;
   <ul>&#60;<font color="darkred">term value</font>="<font color="black"><b>pyridoxal phosphate</b></font>" <font color="darkred">dbxref</font>="<font color="black"><b>COMPOUND:C00018</b></font>" /&#62;
   <br>&#60;<font color="darkred">term value</font>="<font color="black"><b>pyridoxal-5'-phosphate</b></font>" <font color="darkred">dbxref</font>="<font color="black"><b>PDBHET:PLP</b></font>" /&#62;
   </ul>
   &#60;/<font color="darkred">substructure</font>&#62;
 </ul>
 <ul>
 &#60;<font color="darkred">substructure id</font>="<font color="black"><b>BIM000270</b></font>"&#62;
   <ul>&#60;<font color="darkred">bim</font>&#62;<font color="black"><b>(C4A.Pxy)(NZ.Lys)</b></font>&#60;/<font color="darkred">bim</font>&#62;

   <br>&#60;<font color="darkred">term value</font>="<font color="black"><b>N6-pyridoxal phosphate-L-lysine</b></font>" <font color="darkred">dbxref</font>="<font color="black"><b>RESID:AA0119</b></font>" /&#62;
   <br>&#60;<font color="darkred">term value</font>="<font color="black"><b>2-lysine(3-hydroxy-2-methyl-5-phosphonooxymethyl-pyridin-4-ylmethane)</b></font>" <font color="darkred">dbxref</font>="<font color="black"><b>PDBHET:LLP</b></font>" /&#62;
   </ul>
   &#60;/<font color="darkred">substructure</font>&#62;
 </font></ul>...<br><font face="Arial" color="blue">
&#60;/<font color="darkred">term</font>&#62;</font></td>
<td align=right><b>(4.6)</b></td></tr>
</table>

<p>
The summary of relationships is presented in the table:
</p>
<a name="4.7">
<table width="100%">
<tr><td>
<center>
<table cellspacing="3" cellpadding="3">
<tr>
<th bgcolor="white">Relationship</th>
<th bgcolor="white">Inheritance</th>
<th bgcolor="white">Applies to</th>
<th bgcolor="white">Inverse</th>
<tr>
<td bgcolor="white"><font color="green">isKindOf</font></td>
<td bgcolor="white" align=center>all attributes</td>
<td bgcolor="white">PRX to PRX
<br>BIM to BIM
<br>MOL to MOL</td>
<td bgcolor="white" align=right><font color=darkred>includes</font></td>
</tr>

<tr>
<td bgcolor="white"><font color="green">isPartOf</font></td>
<td bgcolor="white" align=center>-</td>
<td bgcolor="white">BIM to BIM
<br>MOL to MOL
<br>MOL to BIM
<br>BIM to PRX</td>
<td bgcolor="white" align=right><font color=darkred>contains</font></td>
</tr>

<tr>
<td bgcolor="white"><font color=green>isBoundTo</font></td>
<td bgcolor="white" align=center>-</td>
<td bgcolor="white">MOL to PRX</td>
<td bgcolor="white" align=right><font color=darkred>binds</font></td>
</tr>
</table>
</center>
</td>
<td align=right><b>(4.7)</b></td>
</tr>
</table>

<a name="references">
<h2>5. References and related documents</h2>

<a name="ref1">
<ol>
<li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=11120676">
Degtyarenko, K. (2000)</a>
Bioinorganic motifs: towards functional classification of metalloproteins.
<a href="http://bioinformatics.oupjournals.org/cgi/content/abstract/16/10/851">
<i>Bioinformatics</i> <b>16</b>, 851-864</a>.

<a name="ref2">

<li>Degtyarenko, K. and Contrino, S. (2004)
COMe: the ontology of bioinorganic proteins.
<a href="http://www.biomedcentral.com/1472-6807/4/3/">
<i>BMC Structural Biology</i> <b>4</b>, 3</a>.

</ol>

</p>
<hr>
<table width="100%" class="tabletop" cellpadding="0">
<tr align="center" class="tablebody">
<td><a href="http://www.flymine.org/come/"><img src="http://www.flymine.org/come/icons/come_s.gif" border="0" alt="COMe back"></a></td>
<td><b>general</b></td>
<td><a href="http://www.flymine.org/come/docs/come_legend.html">relationships</a></td>
<td><a href="http://www.flymine.org/come/docs/centres.html">centres</a></td>
<td><a href="http://www.flymine.org/come/docs/ligands.html">ligands</a></td>
<td><a href="http://www.flymine.org/come/docs/polyhedra.html">coordination polyhedra</a></td>
<td><a href="mailto:kirill@ebi.ac.uk, contrino@ebi.ac.uk">contact</a></i></td>
</tr>
</table>
</font>
</body>
</html>
